cmd/kots/cli/reset-password.go

```diff
@@ -146,3 +151,29 @@ func setKotsadmPassword(password string, namespace string) error {
 
 	return nil
 }
+
+// delete all sessions from the kotsadm-sessions secret
+func deleteAllSessions(namespace string) error {
+	clientset, err := k8sutil.GetClientset()
+	if err != nil {
+		return errors.Wrap(err, "failed to create k8s client")
+	}
+
+	existingSecret, err := clientset.CoreV1().Secrets(namespace).Get(context.TODO(), "kotsadm-sessions", metav1.GetOptions{})
```

Comment:
why do we need to get the secret at all here?

Reply:
never knew there was a `func DeleteCollection` , should have checked the interface.

Reply:
Out of curiousity, what happens if we delete the secret, does it expect a secret to exist to append to, or will the first session create the secret?

Reply:
@trelore if `kotsadm-secret` gets deleted, kots will fallback to environment password value `SHARED_PASSWORD_BCRYPT` 
when someone changes the secret, the new secret get's created